src/Microsoft.DocAsCode.Build.TableOfContents/BuildTocDocument.cs

```diff
@@ -99,5 +99,19 @@ private void BuildCore(TocItemViewModel item, FileModel model, IHostService host
                 }
             }
         }
+
+        #region ISupportIncrementalBuild Members
+
+        public bool CanIncrementalBuild(FileAndType fileAndType)
+        {
+            return true;
+        }
+
+        public string GetIncrementalContextHash()
+        {
+            return null;
```

Comment:
agree with @herohua . the hash would be used to check whether the processor/step can be incremental, currently seems no context info would influence whether the processor can be incremental